aea/cli/common.py

```diff
@@ -222,16 +205,6 @@ def try_to_load_agent_config(ctx: Context, is_exit_on_except: bool = True) -> No
             sys.exit(1)
 
 
-def _load_env_file(env_file: str):
-    """
-    Load the content of the environment file into the process environment.
-
-    :param env_file: path to the env file.
-    :return: None.
-    """
-    load_dotenv(dotenv_path=Path(env_file), override=False)
```

Comment:
see above: now this is potentially needed to the builder. I say "potentially" because the call to this function is commented in `from_aea_proejct`